phpstan.neon

```diff
@@ -1,3 +1,4 @@
 parameters:
+    reportUnmatchedIgnoredErrors: false
```

Comment:
What's the reason behind this? Can't you just remove the ignored errors?

Reply:
The existing "ignoreErrors" options is helpful when running phpstan under php 7.0, as it would throw something like this otherwise:
`TextDocumentItem::$uri (string) does not accept string|null.`
Under php >=7.1 this does not happen anymore, but then you will get the following error instead:
` Ignored error pattern #Property .* does not accept# was not matched in reported errors.`
Which is the reason I have added the `reportUnmatchedIgnoredErrors: false` option.

Reply:
Then I guess a better option would be to pin the PHP version for PHPStan. You can set it using `parameters.phpVersion: 70000` (for 7.0) in `phpstan.neon`.

Reply:
But that doesn't work when running the analysis under `>=7.1`:
` The option 'parameters › phpVersion' expects to be int:70100..80000|null, 70000 given`

Or do you mean creating php version specific phpstan configuration files?

Reply:
Oh, okay. I don't understand the problem _exactly_, but I just think there _has_ to be a better way. But you be the judge of that. : )

Reply:
Well, I agree that adding this option is a bit annoying, but it's only ignoring an error which will occur due to the nature of the different php versions and due to the one specific ignored error which also defined in the same file. So I would say it’s the best solution to add it : )